tools/build-js-documentation.py

```diff
@@ -9,22 +9,38 @@
 in $WORKSPACE/build.
 '''
 
-import os, subprocess, json
+import copy
+import json
+import os
+import subprocess
 
 def main():
     # Workspace is the root of the builder, e.g. "/var/lib/jenkins/workspace/builder".
     workspace = os.getenv('WORKSPACE')
     assert workspace
 
+    # Directory where node/npm binaries have been installed.
+    node_bin = '/var/lib/jenkins/node-v5.11.1-linux-x64/bin'
```

Comment:
I don't understand why we are not adding a symlink in `/usr/local/node/bin` like we have done for go?